ci/prep_cython_cache.sh

```diff
@@ -1,43 +1,73 @@
 #!/bin/bash
 
 ls "$HOME/.cache/"
+
+PYX_CACHE_DIR="$HOME/.cache/pyxfiles"
+pyx_file_list=`find ${TRAVIS_BUILD_DIR} -name "*.pyx"`
+pyx_cache_file_list=`find ${PYX_CACHE_DIR} -name "*.pyx"`
+
 CACHE_File="$HOME/.cache/cython_files.tar"
 
+# Clear the cython cache 0 = NO, 1 = YES
 clear_cache=0
+
+pyx_files=`echo "$pyx_file_list" | wc -l`
+pyx_cache_files=`echo "$pyx_cache_file_list" | wc -l`
```

Comment:
The actually checking is done below. `diff -u $i $PYX_CACHE_DIR${i}` for each pyx file . This code is only here to check no pyx files have been deleted.